packages/nextjs/scaffold.config.ts

```diff
@@ -16,11 +18,13 @@ const scaffoldConfig = {
   // The interval at which your front-end polls the RPC servers for new data
   // it has no effect if you only target the local network (default is 30_000)
   pollingInterval: 30_000,
+  lastConnectedTime: 0,
```

Comment:
Hi, I meant you can just transfer this key into a variable somewhere or maybe in the `constants.ts` file somewhere, not in the scaffold config (this is not a config!). The reason is that if we leave it raw as string, it would be prone to bugs since strings are not enforced if typo. Whereas if you extract it to a constant like I've done below, the linter will tell you that you have a typo and you can have a consistent local storage key should you want to reuse the local storage key.

```ts
const LAST_CONNECTED_TIME_LOCALSTORAGE_KEY = "lastConnectedTime";
```